app/views/events/_form.html.erb

```diff
@@ -55,7 +55,7 @@
         Jede Veranstaltung kann in einer oder mehreren Kategorien abgelegt werden.
       <p>
 
-      <%= render partial: "shared/partials/categories_form", locals: { record: event, record_type: "event", form: f } %>
+      <%= render partial: "shared/partials/categories_form", locals: { record: event, record_type: "event", form: f, tag_filter: "event_record" } %>
```

Comment:
maybe we should update the `record_type` here to `event_record` as well to be in place with all the other base resources?